tests/hikari/impl/test_rest.py

```diff
@@ -2251,7 +2251,7 @@ async def test_edit_channel(
             invitable=True,
             auto_archive_duration=auto_archive_duration,
             flags=12,
-            applied_tags=[channels.ForumTag(name="testing", moderated=True)],
+            applied_tags=[0],
```

Comment:
For better testing, pass a `StubModel(0)` here

Reply:
```suggestion
            applied_tags=[StubModel(0)],
```